File: Python/problem003.py
#Variables
factors = 1
number = 600851475143
prime = 0

#Finds prime factors of 600851475143
while factors != number:
    prime += 1

    #Multiplies number to total if its prime
    if number % prime == 0:
        factors *= prime

#Displays answer
print(prime)  #Expected 6857

File: Java/problem003.java
public class problem003 {
	public static void main(String[] args) {

		//Variables
		long factors = 1L;
		long number = 600851475143L;
		long prime = 0L;

		//Finds prime factors of 600851475143
		while (factors != number) {
			prime += 1;

			//Multiplies number to total if its prime
			if (number % prime == 0L) {
				factors *= prime;
			}
		}

		//Displays answer
		System.out.print(prime); //Expected 6857
	}
}